ember-container-query/package.json

```diff
@@ -21,15 +21,25 @@
   "author": "Isaac J. Lee",
   "exports": {
     ".": "./dist/index.js",
-    "./*": "./dist/*.js",
-    "./addon-main.js": "./addon-main.js"
+    "./*": {
+      "types": "./dist/*.d.ts",
+      "default": "./dist/*.js"
+    },
```

Comment:
Yeah, the `default` key having to be listed last also [caused me trouble](https://github.com/ijlee2/ember-codemod-v1-to-v2/pull/16). Objects shouldn't be used for something with an order dependency...